### ISSUE TITLE
[neural-networks] reproducability of tests

### ISSUE THREAD

**Issue by HenKlei** [HenKlei/pymor#1]
The results obtained by the neural networks should be deterministic, i.e. when running the demo twice, the result should be the same. Maybe it is possible to set a seed in PyTorch to fix the initial weights?

**Comment by HenKlei** [comment]
Done in 7cf247e65b6fd94c0e857bfe7096419b24c82ef3. See also [this note](https://pytorch.org/docs/stable/notes/randomness.html) on reproducability in PyTorch.
